draft-wussler-openpgp-pqc.md

```diff
@@ -804,13 +817,14 @@ SHA3-256 MUST be used to hash the `publicKey` of the recipient.
 For the composite KEM schemes defined in {{kem-alg-specs}} the following
 procedure MUST be used to compute the KEK that wraps a session key. The
 construction is a one-step key derivation function compliant to
-{{NIST-SP800-56C}} Section 4.
+{{NIST-SP800-56C}} Section 4 based on the KMAC256 MAC algorithm defined in
```

Comment:
```suggestion
{{NIST-SP800-56C}} Section 4, based on KMAC256
```

Reply:
Smoothed text a little and inserted a comma for easier reading.

Reply:
I think this still has to be addressed.

Reply:
Sorry, I had missed out on that comment somehow. The suggestion is now adopted.